src/utils/validator.ts

```diff
@@ -18,4 +18,8 @@ const isTimeLimit = (timeLimit: number) => {
   return differenceTime < 2;
 };
 
-export { isEmptyString, isLongerThan, isAllTrue, isTimeLimit };
+const isAllLessThan = (arr: string[], num: number) => {

```

Comment:
```suggestion
const validateElementsLength = (arr: string[], num: number) => {
``` 
또는
```suggestion
const checkMaxLength = (arr: string[], num: number) => {
``` 
요런 네이밍은 어떠신가요?? 

++) ts doc 로 간단한 사용법(+매개변수에 대한 설명)까지 주어진다면 더 가독성이 좋을 거 같아요!

Reply:
check나 validate의 이름은 배열의 `모든 요소`들의 길이가 인자로 받은 수보다 같거나 작은지에 대한 역할을 정확히 보여주지 않다고 생각되어 isAllLessThan을 사용했습니다..!! 해당 파일 내의 함수 이름에 통일성을 주기 위함도 있구요 ㅎㅎ